src/dragonball/src/dbs_pci/src/msix.rs

```diff
@@ -260,7 +260,6 @@ impl MsixState {
         Ok(())
     }
 
-    #[cfg(target_endian = "little")]
```

Comment:
This isn't related to the topic of turning powerpc64le into powerpc64. Also, this is the dragonball VMM here, not kata. This function seems to be part of some PCI device model where endianness usually matters and I strongly suspect this code to be endianness sensitive.

Please drop this change and all the other ones that aren't about powerpc64le.